src/GraphQL.DataLoader.Tests/GraphQL.DataLoader.Tests.csproj

```diff
@@ -18,9 +18,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Bogus" Version="34.0.2" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="3.1.20" />
-    <PackageReference Include="System.Reactive" Version="5.0.0" />
+    <PackageReference Include="Bogus" Version="34.*" />
```

Comment:
I am rather for such a change than against it, although it brings a certain degree of non-determinance in the build.